### DISCUSSION TITLE
Numeric value of constants

### DISCUSSION THREAD

**Discussion by dlinhart14** [JuliaPhysics/PhysicalConstants.jl#27] [Q&A] [answered]
Is there a simple way to obtain the numeric value of constants, or combination of constants? 

I need to multiply the equation by the factor of `e/m_e` and I need the result to be a simpe float without units. Only way I come with is by dividing by the units of that expression `e/m_e/1u"C*kg^-1"` which give me the required factor `1.7588e11` but to get it I need to know the units of the factor which makes the procedure kind of uselles.

**Comment by giordano** [comment, marked as answer]
> Is there a simple way to obtain the numeric value of constants, or combination of constants?

The numerical value of a quantity with units is not well defined, is it?  You can use [`Unitful.ustrip`](https://painterqubits.github.io/Unitful.jl/stable/manipulations/#Unitful.ustrip) instead of dividing by the unit, but you can't get around specifying the unit in which you want your number, otherwise in principle I can give you an arbitrary random number if you ask me for the "numerical value" of a constant without specifying the units.

**Comment by dlinhart14** [comment]
`ustrip` is exactly what I need, thank you!
